### ISSUE TITLE
Thinner version 

### ISSUE THREAD

**Issue by HugoHeneault** [erikflowers/weather-icons#126]
Your icons are really awesome, but sometimes, they are a little to thick. 

It would be great if you were able to create a thinner version! :+1: 

Thanks!

**Comment by ericjames** [comment]
I would also like a font-weight: 100 version where it is a 1 pixel outline!!! Currently is the hot UX flavor of the month

**Comment by HassanSE** [comment]
Agreed. Icons are awesome but thin and ultra light versions are missing.
